packages/xml-views-tooltip/test/tooltip-spec.ts

```diff
@@ -10,61 +10,211 @@ import {
   });
 
-  it("will get hover content UI5 property", () => {
-    const xmlSnippet = `<mvc:View
-                          xmlns:mvc="sap.ui.core.mvc"
-                          xmlns="sap.m"> 
-                          <content>
-                            <List showSeparator⇶s="All"></List>
-                          </content>
-                        </mvc:View>`;
-    const ui5Node = getUI5Node(xmlSnippet, ui5SemanticModel);
-    expectExists(ui5Node, "UI5Node");
-    expect(ui5Node.name).to.equal("showSeparators");
-    expect(ui5Node.kind).equal("UI5Prop");
+  context("hover on attribute key", () => {
+    it("will get hover content UI5 property", () => {
+      const xmlSnippet = `<mvc:View
+                            xmlns:mvc="sap.ui.core.mvc"
+                            xmlns="sap.m"> 
+                            <content>
+                              <List showSeparator⇶s="All"></List>
+                            </content>
+                          </mvc:View>`;
+      const ui5Node = getUI5Node(xmlSnippet, ui5SemanticModel);
+      expectExists(ui5Node, "UI5Node");
+      expect(ui5Node.name).to.equal("showSeparators");
+      expect(ui5Node.kind).equal("UI5Prop");
+    });
+
+    it("will get hover content UI5 property - incorrect key", () => {
+      const xmlSnippet = `<mvc:View
+                            xmlns:mvc="sap.ui.core.mvc"
+                            xmlns="sap.m"> 
+                            <content>
+                              <List showSeparator⇶s1="All"></List>
+                            </content>
+                          </mvc:View>`;
+      const ui5Node = getUI5Node(xmlSnippet, ui5SemanticModel);
+      expect(ui5Node).to.not.exist;
+    });
   });
 
-  it("will get hover content UI5 enum", () => {
-    const xmlSnippet = `<mvc:View
-                          xmlns:mvc="sap.ui.core.mvc"
-                          xmlns="sap.m"> 
-                          <content>
-                            <List showSeparators="Al⇶l"></List>
-                          </content>
-                        </mvc:View>`;
-    const ui5Node = getUI5Node(xmlSnippet, ui5SemanticModel);
-    expectExists(ui5Node, "UI5Node");
-    expect(ui5Node.name).to.equal("All");
-    expect(ui5Node.kind).equal("UI5Enum");
+  context("hover on attribute value", () => {
+    it("will get hover content UI5 enum", () => {
+      const xmlSnippet = `<mvc:View
+                            xmlns:mvc="sap.ui.core.mvc"
+                            xmlns="sap.m"> 
+                            <content>
+                              <List showSeparators="Al⇶l"></List>
+                            </content>
+                          </mvc:View>`;
+      const ui5Node = getUI5Node(xmlSnippet, ui5SemanticModel);
+      expectExists(ui5Node, "UI5Node");
+      expect(ui5Node.name).to.equal("All");
+      expect(ui5Node.kind).equal("UI5EnumValue");
+    });
+
+    it("will get hover content UI5 enum without information", () => {
```

Comment:
It's not an enum, it's a non-enum property (boolean). You can write "UI5 property value for attribute which is not an enum"